regulations/static/regulations/js/source/views/comment/comment-review-view.js

```diff
@@ -107,6 +108,10 @@ var CommentReviewView = Backbone.View.extend({
     this.submitMode();
   },
 
+  back: function() {
+    window.history.back();
+  },
```

Comment:
History functions seem to always need a parens: https://developer.mozilla.org/en-US/docs/Web/API/Window/history

(I also tried out of curiosity, didn't work. :P)